my_list.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] MyList class should implement `#each method` that yields all members of @list in succession. Let's add it